cmd/sifnoded/cmd/tokenregistry.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/server"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	"github.com/spf13/cobra"
+
+	tokenregistrytypes "github.com/Sifchain/sifnode/x/tokenregistry/types"
+)
+
+// SetTokenRegistryAdminCmd set the admin address can update the token registry
+func SetTokenRegistryAdminCmd(defaultNodeHome string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set-genesis-token-registry-admin [address_or_key_name]",
+		Short: "Set a genesis account for token registry to genesis.json",
+		Long: `Set a genesis account to genesis.json. The account has the privilege to update token registry.
+`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+			cdc := clientCtx.Codec
+			serverCtx := server.GetServerContextFromCmd(cmd)
+			config := serverCtx.Config
+			config.SetRoot(clientCtx.HomeDir)
+			addr, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
```

Comment:
Why do we need this?  Can't we just require an address?

Reply:
We need the context, then can write the address into json config file.